doc/design/ice-v1.md

```diff
@@ -12,31 +12,39 @@ We can ignore a whole bunch of things:
 
 1. No resumables or bubblegum used (as all data is available)
 2. All stream input for given customer processed in single run
-3. All stream input for given customer fits in memory
+3. All stream input for given customer fits in memory (actually we do this correctly now)
 4. Nested arrays not supported: if a feature is an array, you cannot do "feature arr ~> latest 5 ~> value :: Array (Array Int)".
+5. Chords are not supported, but we're fast so why not run 100 snapshots
 
 
 Todo
 ----
 
 ## High
-
-1. Melt maps into pair of arrays
-    - (should be simple after melt arrays because majority is done in flatten)
-    - Converting maps to/from C values
-2. Array of strings
-    - Converting to/from C values
-3. Melt bufs (tran)
+1. PSV output for:
+    - Maps, arrays, pairs?
+2. PSV input for arrays of primitive types
+    - Strings, Ints?
+    - Can we have arrays inside structs, or structs inside arrays?
+3. PSV missing output for last item
```

Comment:
The last entity is never processed because it doesn't trigger the entity rollover